client/src/components/Chat.jsx

```diff
@@ -222,6 +222,8 @@ const Chat = () => {
 		}
 		setMessage(e.target.value);
 		adjustTextareaHeight(inputRef);
+		e.target.style.height = '45px';
+		e.target.style.height = `${e.target.scrollHeight}px`;
```

Comment:
please explain your thoughts process here

Reply:
I researched this doing CSS but was not possible it required to adjust the height whenever we type into text area. The scrollheight is height of the element and it changes dynamically as we type and increase the height of text area and we remove the text and decrease the height. I had done this observations before. By setting the text area's height to its scrollHeight, i ensured that it would expand and shrink to fit all of its content without requiring manual adjustment by the user. The initial height is set to 45px using tailwind and when the text area height decreases it should decrease upto 45px so first line is for that.

Reply:
thanks for explaining